pypeit/core/trace_slits.py

```diff
@@ -3085,18 +3088,26 @@ def trace_refine(filt_image, edges, edges_mask, ncoeff=5, npca = None, pca_expla
     trace_model_righ = trace_model + fwhm/2.0
 #    trace_model_left = trace_model - 0.5 #fwhm/2.0
 #    trace_model_righ = trace_model + 0.5 #fwhm/2.0
+
     msgs.info('Extracting filt_image along curved edge traces')
     filt_extract = extract.extract_asymbox2(filt_image, trace_model_left, trace_model_righ)
     if debug:
         ginga.show_image(filt_extract, chname ='rectified filt_image')
 
-    filt_smash_mean, filt_smash_median, filt_smash_sig = sigma_clipped_stats(filt_extract, axis=0, sigma=4.0)
+    # Smash the filtered image
+    #   For instruments where light runs on only a portion of the detector,
+    #   one is recommended to smash only that portion
+    smash_rows = (int(smash_range[0]*nspec), int(smash_range[1]*nspec))
```

Comment:
again naming the variable smash_spec or smash_spat here would be preferred.

Reply:
rows -> spec